MODULE.bazel

```diff
@@ -1,8 +1,24 @@
 module(
     name = "buildbuddy_toolchain",
```

Comment:
We should consider renaming this to `toolchains_buildbuddy`. It's not a hard requirement, but calling a module `toolchains_X` if its purpose is to register toolchains for types it doesn't define itself has become a popular pattern in the BCR.